models/public/wavernn/description/wavernn.md

```diff
@@ -0,0 +1,131 @@
+# wavernn
+
+## Use Case and High-Level Description
+
+WaveRNN is a model for the text-to-speech task originally trained in PyTorch\*
```

Comment:
could you please specify what dataset the model was trained on?

Reply:
Done